src/PolyhedralGeometry/Polyhedron/properties.jl

```diff
@@ -705,6 +705,48 @@ julia> normalized_volume(C)
 normalized_volume(P::Polyhedron) =
   coefficient_field(P)(factorial(dim(P)) * (pm_object(P)).VOLUME)
 
+@doc raw"""
+    castelnuovo_excess(P::Polyhedron)
+
+For an arbitrary lattice polytope, Hibi [Hib94](@cite) proved that the normalized volume is always at least as large as a certain lattice point count.
+This function returns the difference between those two numbers.
+
+# Examples
+```jldoctest
+julia> castelnuovo_excess(cube(4))
+154
+```
+"""
+function castelnuovo_excess(P::Polyhedron)::Integer
+  _assert_lattice(P)
+  d = dim(P)
+  l = P.pm_polytope.N_LATTICE_POINTS::Integer
+  c = P.pm_polytope.N_INTERIOR_LATTICE_POINTS::Integer
```

Comment:
So I just checked and these objects have return type `Polymake.LibPolymake.IntegerAllocated`.

So I suggest this then:
```suggestion
  l = ZZ(pm_object(P).N_LATTICE_POINTS)::ZZRingElem
  c = ZZ(pm_object(P).N_INTERIOR_LATTICE_POINTS)::ZZRingElem
```